clients/mtaclient/retryable_mta_rest_client.go

```diff
@@ -117,7 +117,9 @@ func (c RetryableMtaRestClient) StartMtaOperation(operation models.Operation) (R
 }
 func (c RetryableMtaRestClient) UploadMtaFile(file os.File) (*models.FileMetadata, error) {
 	uploadMtaFileCb := func() (interface{}, error) {
-		return c.mtaClient.UploadMtaFile(file)
+		reopenedFile, _ := os.Open(file.Name())
```

Comment:
error ignore again ...
Who is responsible for closing the file here?

Reply:
Not ignored anymore. The file is closed in the file uploader -> uploadPartFile.